src/components/SetInnerHTML.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class SetInnerHTML extends React.Component {
+
+    render () {
+        return (
+            <div dangerouslySetInnerHTML={{__html: this.props.children}}>
```

Comment:
This seems unnecessary. It looks like this is only used to wrap the h3 in a div. Why not do it directly in MovieDisplay component?